Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -292,6 +298,24 @@ void NetPlayDialog::ConnectWidgets()
           [this] { m_chat_send_button->setEnabled(!m_chat_type_edit->text().isEmpty()); });
 
   // Other
+  connect(m_auto_buffer_box, &QCheckBox::clicked, this, [this](bool value) {
```

Comment:
`&QCheckBox::toggled`

Also please make the bool `checked` instead of `value` for consistency.